src/core/settings/keymap.json

```diff
@@ -111,6 +111,10 @@
     "mac": "command+d",
     "description": "Duplicate selected region"
   },
+  "segment:delete": {
```

Comment:
I am not quite sure what this change does. We moved from `region:remove` to `segment:delete`, but changing it like this doesn't seem like it would fix the problem, more that it would stop all regions from ever being removed by hotkey? 

Maybe you could explain a bit about what was happening after before and after this change. Does this still work in other region removals by the delete key in Image rectangles, or other region based operations?